chain/network/src/network_protocol/proto_conv/handshake.rs

```diff
@@ -0,0 +1,185 @@
+/// Conversion functions for Hanshake messages.
```

Comment:
```suggestion
/// Conversion functions for handshake messages.
```

Reply:
I'd still like to see the "d" though :)

(Personally, I either write things in plain English or in `formatted_code` but I honestly don't care. I only nitpicked here as I saw two things in one word.)

Reply:
> I'd still like to see the "d" though :)

you got me XD